lib/pages/tasks/models/tasks.dart

```diff
@@ -11,18 +11,19 @@ class Tasks {
   static final dbStatus = "status";
   static final dbProjectID = "projectId";
 
-  String title, comment, projectName;
-  int id, dueDate, projectId, projectColor;
-  Status priority;
-  TaskStatus tasksStatus;
-  List<String> labelList = List();
+  String? title, comment, projectName;
+  int? id, dueDate, projectId, projectColor;
+  Status? priority;
+  TaskStatus? tasksStatus;
+  List<String>? labelList = [];
```

Comment:
`List<String>?` will be non-nullable because we are initializing with `[]` list here.
Same goes with `title`,`projectName`,`projectColor`,`projectId`,`dueDate` and `priority`